### PR TITLE
fix(clerk-js): UI fixes on <OrganizationProfile /> to match new design

### PR DESCRIPTION
**Pull request by octoper**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by panteliselef** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @panteliselef, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-7afhtfrvw.clerkstage.dev) | Jan 12, 2024 07:42 PM |

**Comment by octoper** [comment]
!preview

File: packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx [hunk 1]
```diff
@@ -201,12 +186,6 @@ export const RoleSelect = (props: {
             as='span'
             gap={1}
           >
-            <Text
-              as='span'
-              sx={t => ({ color: t.colors.$blackAlpha400 })}
-            >
-              {`${t(localizationKeys('formFieldLabel__role'))}:`}
-            </Text>
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx:209]: Why is the `Role:` word removed here?

Comment (octoper) [reply packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx:209]: As shown in the designs, the Dropdown for the role inside the `<MemberListTable/>`, should not contain he `Role:`. The `Role:` text is only needed when the `<RoleSelect/>` is inside the `<InviteMembersForm/>`, I will need to change that as we use `<RoleSelect/>` in both places.

**Review by anagstef — APPROVED** [review]

File: packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx [hunk 2]
```diff
@@ -111,12 +97,12 @@ const EmptyRow = (props: { localizationKey: LocalizationKey }) => {
...
           localizationKey={props.localizationKey}
-          sx={t => ({
+          sx={{
             margin: 'auto',
             display: 'block',
             width: 'fit-content',
-            fontSize: t.fontSizes.$xs,
-          })}
+          }}
+          variant='body'
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx:105]: Isn't this the default?

File: packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx [hunk 3]
```diff
@@ -163,6 +149,7 @@ export const RoleSelect = (props: {
       elementId='role'
       options={localizedOptions}
       value={value}
+      placeholder='Select role'
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/components/OrganizationProfile/MemberListTable.tsx:152]: Localization here?

File: packages/clerk-js/src/ui/elements/InputGroup.tsx [hunk 4]
```diff
@@ -41,18 +40,28 @@ export const InputGroup = forwardRef<
...
         maxLength={25}
         sx={[
           {
-            borderColor: 'transparent',
+            boxShadow: 'none',
+            border: 'none',
+            ':hover:not(:focus)': {
+              border: 'none',
+              boxShadow: 'none',
+            },
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/elements/InputGroup.tsx:64]: Is is better if we add a new variant for this case?

Comment (octoper) [reply packages/clerk-js/src/ui/elements/InputGroup.tsx:64]: I'm not sure, I think that's only in 2 places.

**Review by anagstef — APPROVED** [review]
Make sure we add the `organizationProfile.invitePage.selectDropdown__role` localization key in the clerk-docs PR here: https://github.com/clerk/clerk-docs/pull/632

File: packages/clerk-js/src/ui/elements/Card/CardRoot.tsx [hunk 5]
```diff
@@ -35,7 +35,6 @@ export const CardRoot = React.forwardRef<HTMLDivElement, CardRootProps>((props,
             width: t.sizes.$100,
             boxShadow: t.shadows.$cardBoxShadow,
             borderRadius: t.radii.$xl,
-            overflow: 'hidden',
```
Comment (anagstef) [inline packages/clerk-js/src/ui/elements/Card/CardRoot.tsx:38]: @octoper is there a reason we removed this? It breaks our footer.

**Pull request merged**